src/Webfactory/Slimdump/SlimdumpCommand.php

```diff
@@ -35,17 +35,21 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $dsn = $input->getArgument('dsn');
+        $noAutoIncrement = $input->getOption('no-auto-increment');
         $db = connect($dsn);
 
         $config = ConfigBuilder::createConfigurationFromConsecutiveFiles($input->getArgument('config'));
-        $this->dump($config, $db, $output);
+        $this->dump($config, $db, $output, $noAutoIncrement);
     }
 
     /**
      * @param Config $config
      * @param Connection $db
+     * @param OutputInterface $output
+     * @param boolean $noAutoIncrement
+     * @throws \Doctrine\DBAL\DBALException
      */
-    public function dump(Config $config, Connection $db, OutputInterface $output)
+    public function dump(Config $config, Connection $db, OutputInterface $output, $noAutoIncrement)
```

Comment:
It's used to query the auto-increment parameter from the table config. Any other suggestion? Passing the boolean directly?